metals-docs/src/main/scala/docs/UserConfigurationModifier.scala

```diff
@@ -14,50 +14,25 @@ class UserConfigurationModifier extends StringModifier {
       code: Input,
       reporter: Reporter
   ): String = {
-    info match {
-      case "lsp-config-default" =>
-        UserConfiguration.options
-          .map { option =>
-            s"""
-               |### ${option.title}
-               |
-               |${option.description}
-               |
-               |**Default**: ${option.default}
-               |
-               |**Example**:
-               |```json
-               |{
-               |  "metals": {
-               |    "${option.key}": ${option.example}
-               |  }
-               |}
-               |```
-               |""".stripMargin
-          }
-          .mkString("\n")
-      case "lsp-config-coc" =>
-        UserConfiguration.options
-          .map { option =>
-            s"""
-               |### ${option.title}
-               |
-               |${option.description}
-               |
-               |**Default**: ${option.default}
-               |
-               |**Example**:
-               |```json
-               |{
-               |  "metals.${option.camelCaseKey}": ${option.example}
-               |}
-               |```
-               |""".stripMargin
-          }
-          .mkString("\n")
-      case els =>
-        reporter.error(s"unknown user-config '$els'")
-        ""
-    }
```

Comment:
So I wasn't fully sure why we actually were displying everything kebab case since looking at the various extensions, everything sends it in via camelcase. So I think it might be better to just align and show them that way.

Reply:
IIRC it used to accept both syntaxes since we originally parsed these from different sources, such as system properties.

Reply:
Totally, it actually still does. I didn't realize before making this change, but you can still pass in all the user configurations as properties.